### PR TITLE
Make link text more descriptive

### PR DESCRIPTION
**Pull request by RobertoJBeltran**
Make link text more descriptive for improved accessibility.

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

**Review by theNaut1lus — CHANGES_REQUESTED** [review]
Add MDN in link title to make the audience know where they are going when they click on it.
so basically from
`click this link to know more` -> `click this link to see the MDN documentation`

something along the lines of this

**Comment by RobertoJBeltran** [comment]
I made a new commit per your suggestion

File: curriculum/challenges/english/03-front-end-libraries/jquery/change-text-inside-an-element-using-jquery.english.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Here's how you would rewrite and emphasize the text of our heading:
 <code>$("h3").html("&#60;em&#62;jQuery Playground&#60;/em&#62;");</code>
 jQuery also has a similar function called <code>.text()</code> that only alters text without adding tags. In other words, this function will not evaluate any HTML tags passed to it, but will instead treat it as the text you want to replace the existing content with.
 Change the button with id <code>target4</code> by emphasizing its text.
-Check this <a href="https://developer.mozilla.org/en/docs/Web/HTML/Element/em" target="_blank">link</a> to know more on the difference between <code>&#60;i&#62;</code> and <code>&#60;em&#62;</code> and their uses.
+Check this link to <a href="https://developer.mozilla.org/en/docs/Web/HTML/Element/em" target="_blank">go to the MDN web docs for <code>&#60;em&#62;</code></a> to learn the difference between <code>&#60;i&#62;</code> and <code>&#60;em&#62</code> and their uses.
```
Comment (RandellDawson) [inline curriculum/challenges/english/03-front-end-libraries/jquery/change-text-inside-an-element-using-jquery.english.md]: @RobertoJBeltran You actually did not make the change @co16353sidak suggested.  Also, after reviewing this myself, I suggest the following change:
```suggestion
<a href="https://developer.mozilla.org/en/docs/Web/HTML/Element/em" target="_blank">View the MDN web docs for &#60;em&#62;</a> to learn the difference between <code>&#60;i&#62;</code> and <code>&#60;em&#62</code> and their uses.
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@RobertoJBeltran Please see my suggested changes.

If you have any questions, please let us know.

File: curriculum/challenges/english/03-front-end-libraries/jquery/change-text-inside-an-element-using-jquery.english.md (hunk 1 above)
Comment (co16353sidak) [reply curriculum/challenges/english/03-front-end-libraries/jquery/change-text-inside-an-element-using-jquery.english.md]: @RobertoJBeltran I agree with @RandellDawson here, this suggestion seems much better to me

**Comment by RobertoJBeltran** [comment]
@RandellDawson  Understood, I will do better next time

**Review by theNaut1lus — APPROVED** [review]
@RandellDawson after the latest commit, everything Is LGTM 🎉 
Ready to be merged

**Review by RandellDawson — APPROVED** [review]

**Review by Manish-Giri — APPROVED** [review]
Hi @RobertoJBeltran 

Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍 We're happy to accept these changes and look forward to future contributions. 📝

**Comment by RobertoJBeltran** [comment]
@Manish-Giri Thank you, I look forward to making more contributions

**Pull request merged**